src/commands/issue.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * This command runs ignite doctor and launches the user's browser
+ * to a new GitHub issue populated with the provided title and doctor command  output
+ */
+import { GluegunToolbox } from "gluegun"
+import {
+  command,
+  p,
+  warning,
+  startSpinner,
+  stopSpinner,
+  prettyPrompt,
+  prefix,
+} from "../tools/pretty"
+const stripANSI = require("strip-ansi")
+
+const isMac = process.platform === "darwin"
+
+const START_CMD = isMac ? "open" : "start"
```

Comment:
Won't work on Linux, but we'll let them fix with their own command.

Reply:
And of course it's ridiculous over there: https://stackoverflow.com/questions/5116473/linux-command-to-open-url-in-default-browser

Reply:
We could use `open` -- https://github.com/sindresorhus/open

Or this command, which won't work with all distros, but get us mostly there: https://stackoverflow.com/a/49013356/204044

But for this PR, let's not worry about it.

Reply:
Ah was hoping there was some cross platform *nix usage with `open` but I guess not